<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditParamDialog</class>
 <widget class="QDialog" name="EditParamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>631</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Machine code</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Worker ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Master code / Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMasterCode">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>208</green>
          <blue>200</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMasterName">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>208</green>
          <blue>200</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMachineCode">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxWorkerID">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSelectMachine">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSelectWorker">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidgetPage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>91</width>
     <height>511</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>MasterData</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Page 0</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButtonUpdate">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>620</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>UPDATE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>620</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>J:/検査画面icon/flash_64.PNG</normaloff>J:/検査画面icon/flash_64.PNG</iconset>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>621</width>
     <height>511</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Global">
    <attribute name="title">
     <string>Global</string>
    </attribute>
    <widget class="QFrame" name="framePanel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>481</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Strategy">
    <attribute name="title">
     <string>Strategy</string>
    </attribute>
    <widget class="QFrame" name="frameFormStrategy">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>484</width>
       <height>484</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Communicate">
    <attribute name="title">
     <string>Communicate</string>
    </attribute>
    <widget class="QFrame" name="frameFormComm">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>484</width>
       <height>484</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
